T3/Gui/OutputUi/FloatDictOutputUi.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Numerics;
+using System.Linq;
+using ImGuiNET;
+using T3.Core.Operator.Slots;
+using T3.Core.DataTypes;
+using T3.Gui;
+using Vector2 = System.Numerics.Vector2;
+
+namespace T3.Gui.OutputUi
+{
+    public class FloatDictOutputUi : OutputUi<Dict<float>>
+    {
+        public override IOutputUi Clone()
+        {
+            return new FloatDictOutputUi()
+            {
+                OutputDefinition = OutputDefinition,
+                PosOnCanvas = PosOnCanvas,
+                Size = Size
+            };
+        }
+
+        private static bool _autoFit = true;
+        private static bool _clear = false;
+        private static float _minFit = -1;
+        private static float _maxFit = 1;
+
+        private static List<List<float>> _previousValues;
+        
+        protected override void DrawTypedValue(ISlot slot)
+        {
+            if (slot is Slot<Dict<float>> typedSlot)
+            {
+                ImGui.BeginChild("##scrolling");
+
+                var v = typedSlot.Value;
+                var outputString = v == null ? "" : string.Join(", ", $"{v:0.000}");
+                ImGui.TextUnformatted($"{outputString}");
+
+                if (v == null)
+                    return;
+
+                var count = Math.Min(1024, v.Count);
```

Comment:
```suggestion
                var trackCount = v.Count.Clamp(0,1024);
```

Reply:
Perhaps `valueCount`? It was called `length` in FloatListOutputUi.cs which I found confusing as well.